pkg/config/config.go

```diff
@@ -238,12 +293,42 @@ const (
 	CPU = "CPU"
 	// CPUDescription is the name of the setting controlling the min and max CPU to same value
 	CPUDescription = "The minimum and maximum CPU a component can consume"
+	// Path indicates path to the binary within current dir or context
+	Path = "Path"
+	// Type indicates type of component source -- git/binary/local
+	Type = "Type"
+	// Ref indicates git ref for the component source
+	Ref = "Ref"
+	// Ports is the space separated list of user specified ports to be opened in the component
+	Ports = "Ports"
+	// App indicates application of which component is part of
+	App = "App"
+	// Project indicates project the component is part of
+	Project = "Project"
+	// ProjectDescription is the description of project component setting
+	ProjectDescription = "Project is the name of the project the component is part of"
+	// AppDescription is the description of app component setting
+	AppDescription = "App is the name of application the component needs to be part of"
+	// PortsDescription is the desctription of the ports component setting
+	PortsDescription = "Ports to be opened in the component"
+	// RefDescription is the description of ref setting
+	RefDescription = "Git ref to use for creating component from git source"
+	// TypeDescription is the description of type setting
+	TypeDescription = "Type of component source - git/binary/local"
+	// PathDescription is the human-readable description of path setting
+	PathDescription = "The path setting indicates to binary in current directory or directory pointed to by context for components created using a binary"
```

Comment:
Needs to be re-worded, perhaps: Path indicates the current git / local directory / binary file.

Reply:
Ack